.gitignore

```diff
@@ -17,6 +17,9 @@
 !/log/.keep
 !/tmp/.keep
 
+# ignore intellij related
+.idea/*
```

Comment:
Reading for a minute or two online, this looks like a common practice in rails as there may be difference in local development environments